build-system-tests/templates/components/react/next/template-package-11.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "react-17-next-11-ts",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@types/node": "^15.12.4",
+    "@types/react": "17.0.47",
+    "@types/react-dom": "17",
+    "next": "^11.1.3",
+    "react": "17",
+    "react-dom": "17",
+    "typescript": "5.0.3"
```

Comment:
Is `typescript` meant to be pinned?

Reply:
Good catch. I'll test and address it in the real PR